src/main/java/IP/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,50 @@
+package IP.duke.command;
+
+import IP.duke.main.DukeException;
+import IP.duke.main.Storage;
+import IP.duke.main.TaskList;
+import IP.duke.main.Ui;
+
+import java.io.IOException;
+
+/**
+ * Represents a command to exit the chatbot.
+ * 
+ * @author Gordon Yit
+ * @version CS2103T, Semester 2
+ */
+public class ExitCommand extends Command {
+    private boolean isExitCommand;
+
+    /**
+     * Class constructor.
+     */
+    public ExitCommand() {
+        isExitCommand = true;
+    }
+
+    /**
+     * Executes the command to exit the chat bot, stores tasks into storage.
+     * 
+     * @param tasks lists of tasks
+     * @param ui the user interface.
+     * @param storage the storage file.
+     * @throws DukeException
```

Comment:
Same as what @eltonyeh mentioned, I think it might be clearer if you add an if... line here!

```suggestion
     * @throws DukeException if...
```